nondex-instrumentation/src/main/java/edu/illinois/nondex/instr/HashMapShufflingAdder.java

```diff
@@ -47,15 +49,23 @@ a copy of this software and associated documentation files (the
     private MethodProperties hasNextProp;
     private MethodProperties nextNodeProp;
 
+    private boolean node;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/gyori/NonDex/file/2674096265/issues/source?bid=3322087&fileBranchId=3333254#l52)